### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kyliel**
refine

### CODE REVIEW

**Comment by kyliel** [comment]
@zikalino could you please review the readme change? If ok, can you merge it? Thank you.
If not, feel free make change correspondingly.

File: README.md [hunk 1]
```diff
@@ -8,61 +8,67 @@ If you use Ansible for Azure resource provisioning purpose, you're strongly enco
...
+   Several reasons for installing Python SDKs are listed here.
 
-    - New module is added to the role and this module is for one new Azure service, which is not part of existing Ansible release yet. Corresponding SDK of this new service needs to be installed.
+   - New module is added to the role and this module is for one new Azure resource, which is not included in existing Ansible releases yet. Corresponding SDK for this new resource needs to be installed.
 
-    - Newer versions of SDKs introduce breaking API change. One specific working version should be installed here.
+   - Newer versions of SDKs may introduce breaking API change. One specific working version should be installed here.
 
-    The required SDKs are listed in the *[~/files/requirements-azure.txt](files/requirements-azure.txt)* file. The tricky part is the installation location, which has to be the same as where existing Azure Python SKDs are installed.
+   The required SDKs are listed in the *[~/files/requirements-azure.txt](files/requirements-azure.txt)* file. The `requirements-azure.txt` file can be found in the installed role folder, which is usually at `~/.ansible/roles/files` folder.
```
Comment (yungezz) [inline README.md]: usaually -> by default

File: README.md [hunk 2]
```diff
@@ -8,61 +8,67 @@ If you use Ansible for Azure resource provisioning purpose, you're strongly enco
...
 
-      ``` bash
-      $ pip show azure-mgmt-storage
-      ```
+  ``` bash
+  $ pip show azure-mgmt-storage
+  ```
 
-    The output can be like below. 
+   You can get the output like below.
```
Comment (yungezz) [inline README.md]: old version sounds better

**Comment by yuwzho** [comment]
The change is  already in the master branch

**Pull request closed without merging**